src/main/java/io/tarantool/driver/api/space/options/proxy/ProxyBucketIdOptions.java

```diff
@@ -0,0 +1,39 @@
+package io.tarantool.driver.api.space.options.proxy;
+
+import io.tarantool.driver.api.space.options.OperationWithBucketIdOptions;
+
+import java.util.Optional;
+
+/**
+ * Represent options for functions that uses bucket id to find storage location.
+ *
+ * @author Alexey Kuzin
+ * @author Artyom Dubinin
+ */
+abstract class ProxyBucketIdOptions<B extends ProxyBucketIdOptions<B>> extends ProxyBaseOptions<B>
+        implements OperationWithBucketIdOptions {
+
+    public static final String BUCKET_ID = "bucket_id";
+
+    /**
+     * Specifies bucket id for operation to perform it on storage with this bucket.
+     * By default, crud extracts a priority key from a tuple, processes it through strcrc32 hash function
```

Comment:
```suggestion
     * By default, crud extracts primary key from a tuple and calculates an strcrc32 hash function value
```

`strcrc32` or `mpcrc32`? I doubt that the former one.

Reply:
Still strcrc32 https://github.com/tarantool/crud/blob/d6f7666289321e7b14304d9928dad21ff687cf92/crud/common/sharding/init.lua#L45

Reply:
Fixed